### DISCUSSION TITLE
Different Visuals

### DISCUSSION THREAD

**Discussion by nomadnomaiden** [agronick/aa-torque#117] [Q&A]
Hi everyone,

Thank you very much agronick for the great app. I wonder is it possible to use different background images and themes for different dashboards and different gauge images in one dashboard. That would be cool for instance if we select images under dashboards > data element settings menu instead of main menu. I want to create some thematic visuals (i.e. gauge PNG s that are not same but alike) and use them in some combinations but not sure if it is doable with some code altearations.

**Comment by agronick** [comment]
Yeah thats doable. The theme can't change without reloading the view but the bg image can. I can look at moving it to the dashboard.

**Reply by nomadnomaiden** [reply]
I guess you mean opening-closing dashboard menu or changing an element in it by "realoading the view". Btw I just realised "Theme" stands for the combination of 3 main elements : Gauge image with ticks, gauge image without ticks and needle image.  So we can already use 2 different gauge images in one dashboard thanks to "tick values" option . A third variation would be great since the app layout contains 3 gauges . Actually I wish every individual gauge area would have its own setting, a set of circle image and needle image selectable in their dropdown menu. And all these 3 gauges with fonts and background could be saved under a "Theme" package created and named by user. In short what I mean is a theme should be a package of elements, usable for one dashboard. But all those seems to be a lot of work '^_^ Just an idea if you like and decide to look into.

I am working on things seem like this so all that fuss coming from it hehehe. Not finished tho.

![WhatsApp Image 2024-02-26 at 14 27 15](https://github.com/agronick/aa-torque/assets/159882775/72f5cc5d-2849-4774-8629-596a2bdb5ae4)
![WhatsApp Image 2024-02-26 at 14 26 46](https://github.com/agronick/aa-torque/assets/159882775/e41ebe94-036c-4652-9be2-3277cd1985b4)

Wanted to change prev-next arrows and chart icon but I see these are just code based which is a subject I am a caveman when it comes to it :D
